<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWidget</class>
 <widget class="QWidget" name="GameWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GameWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 255, 255);
background-color: qlineargradient(spread:reflect, x1:0.09, y1:0.096, x2:0.977, y2:0.939101, stop:0 rgba(15, 111, 17, 255), stop:1 rgba(16, 166, 149, 255));</string>
  </property>
  <widget class="QPushButton" name="playAgainButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>10</y>
     <width>131</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>playAgainButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="endBtn">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>80</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>endBtn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="scoreBtn">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>40</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>scoreBtn</string>
   </property>
  </widget>
  <widget class="QLabel" name="scoreLabel">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>120</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDealer1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>100</width>
     <height>170</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelDealer2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>100</width>
     <height>170</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelDealer3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>100</width>
     <height>170</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelDealer4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>100</width>
     <height>170</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelDealer5">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>20</y>
     <width>100</width>
     <height>170</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelDealer6">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>20</y>
     <width>100</width>
     <height>170</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelPlayer6">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>380</y>
     <width>100</width>
     <height>170</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelPlayer3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>380</y>
     <width>100</width>
     <height>170</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelPlayer4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>380</y>
     <width>100</width>
     <height>170</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelPlayer2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>380</y>
     <width>100</width>
     <height>170</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelPlayer1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>100</width>
     <height>170</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelPlayer5">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>380</y>
     <width>100</width>
     <height>170</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelPlayerScore">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>410</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 14pt &quot;Segoe UI&quot;;
color: rgb(255, 0, 4);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="midLineWidth">
    <number>-4</number>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelDealerScore">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>150</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 14pt &quot;Segoe UI&quot;;
color: rgb(255, 0, 4);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="midLineWidth">
    <number>-4</number>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelBackSide">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>200</y>
     <width>100</width>
     <height>170</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
